cellpack/autopack/loaders/recipe_loader.py

```diff
@@ -132,6 +151,31 @@ def _migrate_ingredient(old_ingredient):
                 new_ingredient[v1_to_v2_name_map[attribute]] = old_ingredient[attribute]
             elif attribute in unused_attributes_list:
                 del old_ingredient[attribute]
+            elif attribute in convert_to_partners_map:
+                if "partners" not in new_ingredient:
+                    partners = {}
+                    new_ingredient["partners"] = partners
+                partners[convert_to_partners_map[attribute]] = old_ingredient[attribute]
+
+        if (
+            "orientBiasRotRangeMax" in old_ingredient
+            or "orientBiasRotRangeMin" in old_ingredient
+        ):
+            range_min = (
+                old_ingredient["orientBiasRotRangeMin"]
+                if "orientBiasRotRangeMin" in old_ingredient
+                else -pi
+            )
+            range_max = (
+                old_ingredient["orientBiasRotRangeMax"]
+                if "orientBiasRotRangeMax" in old_ingredient
+                else pi
+            )
+            new_ingredient["orient_bias_range"] = [range_min, range_max]
```

Comment:
you could change this to a function that returns  `[range_min, range_max]`

ie:
```python
def _convert_rotation_range(old_ingredient):
            range_min = (
                old_ingredient["orientBiasRotRangeMin"]
                if "orientBiasRotRangeMin" in old_ingredient
                else -pi
            )
            range_max = (
                old_ingredient["orientBiasRotRangeMax"]
                if "orientBiasRotRangeMax" in old_ingredient
                else pi
            )
            return [range_min, range_max]

# lines 164 - 174:
new_ingredient["orient_bias_range"] = RecipeLoader._convert_rotation_range(old_ingredient)
```